src/comms.cpp

```diff
@@ -174,29 +174,31 @@ void wallPlate_Emulation() {
     // wait up to 15 seconds to look for an existing wallplate or it could be booting, so need to wait
 	if (currentMillis - serialDetected < 15000 || wallplateBooting == true) {
 		if (currentMillis - lastRequestMillis > 1000) {
-			Serial.println("Looking for security+ 1.0 wall panel...");
+			Serial.println("Looking for security+ 1.0 DIGITAL wall panel...");
```

Comment:
These log messages won't log over the network.  Should consider using the macros instead.

Reply:
yea i know... could change, but it was that way before i touched the code for the sec 1 fix
let me change it, then @dkerr64  figure out how to squash commits :)